Calculator/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string xInput, yInput;
+            double x, y;
+            
+            Console.WriteLine("Input x:");
+            xInput = Console.ReadLine();
+            while (!double.TryParse(xInput, out x))
```

Comment:
@twinkleyarns Block from 14 to 18 looks like From 22 to 26, maybe it could be refactoed?